backend/Controllers/UserController.cs

```diff
@@ -0,0 +1,66 @@
+using Microsoft.AspNetCore.Mvc;
+using System;
+using WhaleSpotting.Services;
+using WhaleSpotting.Helpers;
+using WhaleSpotting.Models.Request;
+using WhaleSpotting.Models.Database;
+using WhaleSpotting.Models.Response;
+
+namespace WhaleSpotting.Controllers 
+{
+    [ApiController]
+    [Route("/users")]
+    public class UserController : ControllerBase 
+    {
+        private readonly IAuthService _authService;
+        private readonly IUserService _userService;
+
+        public UserController
+        (
+            IAuthService authService,
+            IUserService userService
+        ) {
+            _authService = authService;
+            _userService = userService;
+        }
+
+        [HttpPost("create")]
+        public ActionResult<CreateUserResponse> Create([FromHeader] string authorization, [FromBody] CreateUserRequest newUserRequest)
```

Comment:
As per below, you can just return a `User` object
```suggestion
        public ActionResult<User> Create([FromHeader] string authorization, [FromBody] CreateUserRequest newUserRequest)
```